content/source/docs/cloud/api/workspace-resources.html.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: "cloud"
+page_title: "Workspaces - API Docs - Terraform Cloud and Terraform Enterprise"
```

Comment:
```suggestion
page_title: "Workspace Resources - API Docs - Terraform Cloud and Terraform Enterprise"
```

Reply:
Oh thanks!  Forgot to change that after reverting it back to no longer being nested.  Good 👁️